src/config.rs

```diff
@@ -712,10 +718,15 @@ pub struct DbConfig {
     pub use_direct_io_for_flush_and_compaction: bool,
     pub enable_pipelined_write: bool,
     pub enable_unordered_write: bool,
+    #[config(submodule)]
     pub defaultcf: DefaultCfConfig,
+    #[config(submodule)]
     pub writecf: WriteCfConfig,
+    #[config(submodule)]
     pub lockcf: LockCfConfig,
+    #[config(submodule)]
     pub raftcf: RaftCfConfig,
+    #[config(skip)]
```

Comment:
why not include Titan

Reply:
Do we now support change Titan config dynamically?

Reply:
`blob_run_mode` can be changed dynamically